dspace-api/src/main/java/org/dspace/handle/dao/impl/HandleDAOImpl.java

```diff
@@ -83,7 +83,7 @@ public long countHandlesByPrefix(Context context, String prefix) throws SQLExcep
     @Override
     public int updateHandlesWithNewPrefix(Context context, String newPrefix, String oldPrefix) throws SQLException
     {
-        String hql = "UPDATE handle set handle = concat(:newPrefix,'/',id WHERE handle like :oldPrefix%";
+        String hql = "UPDATE Handle set handle = concat(:newPrefix,'/',handle_id) WHERE handle like concat(:oldPrefix,'%')";
```

Comment:
Not sure but the use of handle_id here looks strange to me.
I have also found this post on stackoverflow
http://stackoverflow.com/questions/27345133/use-column-name-instead-of-entity-property-in-jpql